app/data/models/form-two.js

```diff
@@ -0,0 +1,94 @@
+module.exports = (sequelize, DataTypes) => {
+  const formTwo = sequelize.define('form_two', {
+    id: {
+      autoIncrement: true,
+      autoIncrementIdentity: true,
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true
+    },
+    dog_id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      references: {
+        model: 'dog',
+        key: 'id'
+      }
+    },
+    registration_id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      references: {
+        model: 'registration',
+        key: 'id'
+      }
+    },
+    form_two_submitted: {
+      type: DataTypes.DATE,
+      allowNull: true
+    },
+    submitted_by_id: {
```

Comment:
I assume we're doing soft-deletes on user-account so that an FK to the id is ok here? Otherwise, we'll just have to store the email and not use an FK